lib/route/neigh.c

```diff
@@ -511,6 +520,9 @@ static void neigh_dump_line(struct nl_object *a, struct nl_dump_params *p)
 	if (n->ce_mask & NEIGH_ATTR_VLAN)
 		nl_dump(p, "vlan %d ", n->n_vlan);
 
+	if (n->ce_mask & NEIGH_ATTR_NHID)
+		nl_dump(p, "nhid %d ", n->n_nhid);
```

Comment:
needs `"%u"` because `n->n_nhid` is `uint32_t`.

Technically, in plain C it might be that `uint32_t` is not the same as `unsigned int`, and thus `"%u"` would not be right. However, we rely on `sizeof(unsigned) >= sizeof(uint32_t)`, so `"%u"` is actually correct. To my knowledge, Linux kernel requires that `int` is at least 32bit(??).

Reply:
done